<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camera_ui</class>
 <widget class="QWidget" name="camera_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
     <item>
      <widget class="QLabel" name="image_resolution_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>(0, 0) - </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="camer_frame_view">
       <property name="minimumSize">
        <size>
         <width>480</width>
         <height>360</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="setting_groupbox">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>625</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>2D 關節點</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QGroupBox" name="camera_setting_groupbox">
        <property name="title">
         <string>相機設定</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="fps_label">
             <property name="text">
              <string>FPS:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="camera_id_label">
               <property name="text">
                <string>相機ID:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="camera_id_input">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QPushButton" name="open_camera_btn">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>開啟相機</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="start_code_btn">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>開始分析</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="kpt_adjust_groupbox">
        <property name="title">
         <string>手動關節點修正</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QCheckBox" name="show_skeleton_checkBox">
             <property name="text">
              <string>Show Skeleton</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="show_bbox_checkbox">
             <property name="text">
              <string>Show BBox</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="show_line_checkBox">
             <property name="text">
              <string>Show Line</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="keypoint_table">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <column>
            <property name="text">
             <string>Keypoint</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>X</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Y</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>有無更改</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
